rasa/shared/core/domain.py

```diff
@@ -327,7 +327,21 @@ def _transform_intent_properties_for_internal_use(
         """
         name, properties = list(intent.items())[0]
```

Comment:
Could you please also update the docstring for this function? 

It currently says the following: `intent: The intents as provided by a domain file.`
In fact, `intent` is just a single intent, not many of them. The current docstring is pretty confusing.